src/main/kotlin/com/viartemev/requestmapper/GoToRequestMappingAction.kt

```diff
@@ -2,19 +2,29 @@ package com.viartemev.requestmapper
 
 import com.intellij.ide.actions.GotoActionBase
 import com.intellij.ide.util.gotoByName.ChooseByNamePopup
+import com.intellij.navigation.ChooseByNameContributor
 import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.actionSystem.CommonDataKeys.PROJECT
+import com.intellij.openapi.extensions.ExtensionPointName
 import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.project.Project
+
+private const val EXTENSION_POINT_NAME = "com.viartemev.requestmapper.requestMappingContributor"
 
 class GoToRequestMappingAction : GotoActionBase(), DumbAware {
 
     override fun gotoActionPerformed(e: AnActionEvent) {
         val project = e.getData(PROJECT) ?: return
 
-        val requestMappingModel = RequestMappingModel(project)
+        val requestMappingModel = getRequestMappingModel(project)
         showNavigationPopup(e, requestMappingModel, GoToRequestMappingActionCallback(), null, true, false)
     }
 
+    fun getRequestMappingModel(project: Project): RequestMappingModel {
```

Comment:
Will this method be called once? 
If yes, it's ok to use this approach, on the opposite we will request contributors all time.
Should we extract the logic to object class and call it from any place?
In this case, we can use this object instead of searching for action in `RequestMappingGoToContributor `
P.S. I'm not sure how many times this method will be called.

Reply:
Fixed. You are right. Thank you!